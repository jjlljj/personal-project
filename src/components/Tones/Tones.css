.tones-section {
  display: flex;
  margin: 30px 0;
  justify-content: center;
}

.tones-wrapper  {
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
  margin: 20px;
}
