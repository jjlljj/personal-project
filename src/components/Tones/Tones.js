import React from 'react'
import './Tones.css'
import { tonesList } from '../../data/tonesData'
import ToneCard from  '../ToneCard/ToneCard'

export const Tones = () => {
  const renderedCards = tonesList.map(tone => {
    return (
      <ToneCard {...tone} key={tone+'card'}/>
    )
  })
  
  return (
    <div className="tones-section">
      <div className="tones-wrapper">
        { renderedCards }
      </div>
    </div>
  )
}

export default Tones