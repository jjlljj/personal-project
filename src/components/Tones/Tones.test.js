/* eslint-disable */
import React from 'react'
import { shallow } from 'enzyme'
import { Tones } from './Tones'

describe('Tones', () => {
  let renderedComponent

  beforeEach(() => {
    renderedComponent = shallow(<Tones />)
  })

  it('should match snapshot', () => {
    expect(renderedComponent).toMatchSnapshot()
  })
})
