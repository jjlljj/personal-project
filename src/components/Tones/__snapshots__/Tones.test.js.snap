// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tones should match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="tones-wrapper"
>
        <Unknown
                description="analytical is a tone which is productive in  an dbblah blah"
                tone_id="analytical"
                tone_name="Analytical"
        />
        <Unknown
                description="positive is a tone which is  in negotionatinos an dbblah blah"
                tone_id="positive"
                tone_name="Positive"
        />
        <Unknown
                description="confident is a tone which is  in  an dbblah blah"
                tone_id="confident"
                tone_name="Confident"
        />
        <Unknown
                description="tentative is a tone which is in negotionatinos an dbblah blah"
                tone_id="tentative"
                tone_name="Tentative"
        />
        <Unknown
                description="fear is a tone which is  i an dbblah blah"
                tone_id="fear"
                tone_name="Fear"
        />
        <Unknown
                description="sadness is a tone which is in an dbblah blah"
                tone_id="sadness"
                tone_name="Sadness"
        />
        <Unknown
                description="anger is a tone which is  in negotionatinos an dbblah blah"
                tone_id="anger"
                tone_name="Anger"
        />
        <Unknown
                description="A neutral tone has no strong tones associated with it"
                tone_id="neutral"
                tone_name="Neutral"
        />
        <Unknown
                description="A disgusted tone has no tones associated with it"
                tone_id="disgust"
                tone_name="Disgust"
        />
</div>,
      "className": "tones-section",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <Unknown
            description="analytical is a tone which is productive in  an dbblah blah"
            tone_id="analytical"
            tone_name="Analytical"
/>,
          <Unknown
            description="positive is a tone which is  in negotionatinos an dbblah blah"
            tone_id="positive"
            tone_name="Positive"
/>,
          <Unknown
            description="confident is a tone which is  in  an dbblah blah"
            tone_id="confident"
            tone_name="Confident"
/>,
          <Unknown
            description="tentative is a tone which is in negotionatinos an dbblah blah"
            tone_id="tentative"
            tone_name="Tentative"
/>,
          <Unknown
            description="fear is a tone which is  i an dbblah blah"
            tone_id="fear"
            tone_name="Fear"
/>,
          <Unknown
            description="sadness is a tone which is in an dbblah blah"
            tone_id="sadness"
            tone_name="Sadness"
/>,
          <Unknown
            description="anger is a tone which is  in negotionatinos an dbblah blah"
            tone_id="anger"
            tone_name="Anger"
/>,
          <Unknown
            description="A neutral tone has no strong tones associated with it"
            tone_id="neutral"
            tone_name="Neutral"
/>,
          <Unknown
            description="A disgusted tone has no tones associated with it"
            tone_id="disgust"
            tone_name="Disgust"
/>,
        ],
        "className": "tones-wrapper",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "Analyticalcard",
          "nodeType": "function",
          "props": Object {
            "description": "analytical is a tone which is productive in  an dbblah blah",
            "tone_id": "analytical",
            "tone_name": "Analytical",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "Positivecard",
          "nodeType": "function",
          "props": Object {
            "description": "positive is a tone which is  in negotionatinos an dbblah blah",
            "tone_id": "positive",
            "tone_name": "Positive",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "Confidentcard",
          "nodeType": "function",
          "props": Object {
            "description": "confident is a tone which is  in  an dbblah blah",
            "tone_id": "confident",
            "tone_name": "Confident",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "Tentativecard",
          "nodeType": "function",
          "props": Object {
            "description": "tentative is a tone which is in negotionatinos an dbblah blah",
            "tone_id": "tentative",
            "tone_name": "Tentative",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "Fearcard",
          "nodeType": "function",
          "props": Object {
            "description": "fear is a tone which is  i an dbblah blah",
            "tone_id": "fear",
            "tone_name": "Fear",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "Sadnesscard",
          "nodeType": "function",
          "props": Object {
            "description": "sadness is a tone which is in an dbblah blah",
            "tone_id": "sadness",
            "tone_name": "Sadness",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "Angercard",
          "nodeType": "function",
          "props": Object {
            "description": "anger is a tone which is  in negotionatinos an dbblah blah",
            "tone_id": "anger",
            "tone_name": "Anger",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "Neutralcard",
          "nodeType": "function",
          "props": Object {
            "description": "A neutral tone has no strong tones associated with it",
            "tone_id": "neutral",
            "tone_name": "Neutral",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": "Disgustcard",
          "nodeType": "function",
          "props": Object {
            "description": "A disgusted tone has no tones associated with it",
            "tone_id": "disgust",
            "tone_name": "Disgust",
          },
          "ref": null,
          "rendered": null,
          "type": [Function],
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="tones-wrapper"
>
          <Unknown
                    description="analytical is a tone which is productive in  an dbblah blah"
                    tone_id="analytical"
                    tone_name="Analytical"
          />
          <Unknown
                    description="positive is a tone which is  in negotionatinos an dbblah blah"
                    tone_id="positive"
                    tone_name="Positive"
          />
          <Unknown
                    description="confident is a tone which is  in  an dbblah blah"
                    tone_id="confident"
                    tone_name="Confident"
          />
          <Unknown
                    description="tentative is a tone which is in negotionatinos an dbblah blah"
                    tone_id="tentative"
                    tone_name="Tentative"
          />
          <Unknown
                    description="fear is a tone which is  i an dbblah blah"
                    tone_id="fear"
                    tone_name="Fear"
          />
          <Unknown
                    description="sadness is a tone which is in an dbblah blah"
                    tone_id="sadness"
                    tone_name="Sadness"
          />
          <Unknown
                    description="anger is a tone which is  in negotionatinos an dbblah blah"
                    tone_id="anger"
                    tone_name="Anger"
          />
          <Unknown
                    description="A neutral tone has no strong tones associated with it"
                    tone_id="neutral"
                    tone_name="Neutral"
          />
          <Unknown
                    description="A disgusted tone has no tones associated with it"
                    tone_id="disgust"
                    tone_name="Disgust"
          />
</div>,
        "className": "tones-section",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <Unknown
              description="analytical is a tone which is productive in  an dbblah blah"
              tone_id="analytical"
              tone_name="Analytical"
/>,
            <Unknown
              description="positive is a tone which is  in negotionatinos an dbblah blah"
              tone_id="positive"
              tone_name="Positive"
/>,
            <Unknown
              description="confident is a tone which is  in  an dbblah blah"
              tone_id="confident"
              tone_name="Confident"
/>,
            <Unknown
              description="tentative is a tone which is in negotionatinos an dbblah blah"
              tone_id="tentative"
              tone_name="Tentative"
/>,
            <Unknown
              description="fear is a tone which is  i an dbblah blah"
              tone_id="fear"
              tone_name="Fear"
/>,
            <Unknown
              description="sadness is a tone which is in an dbblah blah"
              tone_id="sadness"
              tone_name="Sadness"
/>,
            <Unknown
              description="anger is a tone which is  in negotionatinos an dbblah blah"
              tone_id="anger"
              tone_name="Anger"
/>,
            <Unknown
              description="A neutral tone has no strong tones associated with it"
              tone_id="neutral"
              tone_name="Neutral"
/>,
            <Unknown
              description="A disgusted tone has no tones associated with it"
              tone_id="disgust"
              tone_name="Disgust"
/>,
          ],
          "className": "tones-wrapper",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": "Analyticalcard",
            "nodeType": "function",
            "props": Object {
              "description": "analytical is a tone which is productive in  an dbblah blah",
              "tone_id": "analytical",
              "tone_name": "Analytical",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "Positivecard",
            "nodeType": "function",
            "props": Object {
              "description": "positive is a tone which is  in negotionatinos an dbblah blah",
              "tone_id": "positive",
              "tone_name": "Positive",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "Confidentcard",
            "nodeType": "function",
            "props": Object {
              "description": "confident is a tone which is  in  an dbblah blah",
              "tone_id": "confident",
              "tone_name": "Confident",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "Tentativecard",
            "nodeType": "function",
            "props": Object {
              "description": "tentative is a tone which is in negotionatinos an dbblah blah",
              "tone_id": "tentative",
              "tone_name": "Tentative",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "Fearcard",
            "nodeType": "function",
            "props": Object {
              "description": "fear is a tone which is  i an dbblah blah",
              "tone_id": "fear",
              "tone_name": "Fear",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "Sadnesscard",
            "nodeType": "function",
            "props": Object {
              "description": "sadness is a tone which is in an dbblah blah",
              "tone_id": "sadness",
              "tone_name": "Sadness",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "Angercard",
            "nodeType": "function",
            "props": Object {
              "description": "anger is a tone which is  in negotionatinos an dbblah blah",
              "tone_id": "anger",
              "tone_name": "Anger",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "Neutralcard",
            "nodeType": "function",
            "props": Object {
              "description": "A neutral tone has no strong tones associated with it",
              "tone_id": "neutral",
              "tone_name": "Neutral",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "Disgustcard",
            "nodeType": "function",
            "props": Object {
              "description": "A disgusted tone has no tones associated with it",
              "tone_id": "disgust",
              "tone_name": "Disgust",
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
